CircleciScripts/framework_list.py

```diff
@@ -3,24 +3,27 @@
 # consume for the release process. Packages toward the bottom of the list
 # depend on packages toward the top of the list.
 
+# Note that this list isn't a comprehensive list of Xcode schemas or targets
+# that need to be built and tested, only a model of dependencies for cocoapods.
+
 grouped_frameworks = [
     # No dependencies
     [
         'AWSCore',
         'AWSCognitoIdentityProviderASF',
     ],
 
-    # Depends only on AWSCognitoIdentityProviderASF
-    ['AWSCognitoAuth'],
+    [
```

Comment:
so this regrouping works? any way to test this?

Reply:
I've been trying to think of a way to test without a full release, and all I can think of is trying to push to a local repo. However, that doesn't really test the timing and dependencies very well.